Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Keep version in-sync with .travis.yml, .mono_repo.yml and app/lib/shared/versions.dart
```

Comment:
Can we reduce the number of sources of truth?

Reply:
I'm not sure about `.mono_repo.yaml` (if it could infer things), but Dockerfile and `.travis.yaml` seem to be independent of anything and can't take it as an external variable.